frontend/src/app/Settings/ManageOrganizationContainer.tsx

```diff
@@ -33,10 +33,7 @@ const SET_ORGANIZATION = gql`
 `;
 
 const ManageOrganizationContainer: any = () => {
-  const { data, loading, error } = useQuery<Data, {}>(GET_ORGANIZATION, {
```

Comment:
Where did the GET_ORGANIZATION end up in the new code? Do we get that from the whoami endpoint?

Reply:
The only purpose of this page, at least right now, is to update the organization name. 

Right now when the app loads, a query to whoami is called, which has the organization name. `GET_ORGANIZATION` just makes a query again for the org name.

In this PR, the whoamI result is stored in redux context and re-used here, so the `GET_ORGANIZATION` query becomes redundant. 

Long run, I can see this moving to an admin page; or hidden away behind a role-based access